src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/filters/table/TableFilter.java

```diff
@@ -0,0 +1,10 @@
+package io.stargate.sgv2.jsonapi.service.operation.model.impl.filters.table;
+
+import io.stargate.sgv2.jsonapi.service.operation.model.impl.filters.DBFilterBase;
+
+public abstract class TableFilter extends DBFilterBase {
+  // TODO- the path is the column name here, maybe rename ?
```

Comment:
Interesting question: since it's both "path" (in logical in-memory presentation) and actual "column" it maps to/from, could consider either.